File: 1238.py
test_case = int(input())
for i in range(test_case):
    word1, word2 = input().split(' ')
    final_words = []
    count = 0
    for i, j in zip(word1, word2):
        final_words.append(i)
        final_words.append(j)
        count = count + 1
    if(len(word1) > len(word2)):
        final_words.append(word1[count:])
    if(len(word2) > len(word1)):
        final_words.append(word2[count:])
    print("".join(final_words))

File: 1238.c
#include <stdio.h>
int main()
{
    char Word1[50], Word2[50];
    int test_case, a, b, i, j;
    scanf("%d", &test_case);
    for (i = 0; i < test_case; i++)
    {
        scanf("%s", &Word1);
        scanf("%s", &Word2);
        for (a = 0, b = 0, j = 0; j < 50; j++)
        {
            if (Word1[j] == '\0')
            {
                a = 1;
            }
            if (Word2[j] == '\0')
            {
                b = 1;
            }
            if (a == 1 && b == 1)
            {
                break;
            }
            if (a == 0)
            {
                printf("%c", Word1[j]);
            }
            if (b == 0)
            {

                printf("%c", Word2[j]);
            }
        }
        printf("\n");
    }
}